runners/direct-java/pom.xml

```diff
@@ -77,95 +163,6 @@
         </executions>
       </plugin>
 
-      <!-- Ensure that the Maven jar plugin runs before the Maven
-        shade plugin by listing the plugin higher within the file. -->
-      <plugin>
```

Comment:
surefire runs in the test phase (http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html)
jar runs in the package phase (https://maven.apache.org/plugins/maven-jar-plugin/jar-mojo.html)

test phase runs before package phase (http://maven.apache.org/ref/3.5.0/maven-core/lifecycles.html#default_Lifecycle)

The order in which the plugins execute first depends on the maven lifecycle and then if two or more plugins are within the same phase then the order they are defined within the file matters.within the same phase